addons/mail/controllers/discuss/channel.py

```diff
@@ -38,6 +38,15 @@ def discuss_channel_add_guest_as_member(self, channel_id, channel_uuid):
             except UserError:
                 raise NotFound()
 
+    @http.route('/discuss/channel/update_avatar', methods=['POST'], type='json', auth='public')
+    def discuss_channel_avatar_update(self, channel_id, data):
+        channel = request.env['discuss.channel'].browse(int(channel_id)).exists()
+        if not channel or not data:
+            raise NotFound()
+        channel.write({
+            'image_128': data,
+        })
```

Comment:
We started formatting the new Python code with the Black code formatter, using the default configuration except that the line length was set to 120. The whole ChannelController was formatted this way, so you may want to update your code accordingly 🙂